PyTado/interface.py

```diff
@@ -355,6 +366,23 @@ def setZoneOverlay(self, zone, overlayMode, setTemp=None, duration=None, deviceT
         data = self._apiCall(cmd, "PUT", post_data)
         return data
 
+    def setHome(self):
+        """Sets HomeState to HOME """
+        # it seems, this can be set anytime
```

Comment:
Nope, the API won't let me do it:

```
{
    "errors": [
        {
            "code": "unableToUpdateHomePresence",
            "title": "Tried to update to AWAY though there is at least one mobile device detected inside the home fence"
        }
    ]
}
```
(It seems the `title` is wrong/swapped (when setting HOME the API says something about AWAY and vice-versa)